client/src/components/AddPollDialog.js

```diff
@@ -90,6 +90,7 @@ class AddPollDialog extends Component {
             image2: "",
             expiresOn: "",
             sendToList: "",
+            selectedList: "",
```

Comment:
this key is declared but not used anywhere

Reply:
Removed it.